Makefile

```diff
@@ -110,6 +110,42 @@ fmt refmt::
 fmt-no-promote refmt-no-promote::
 	@esy dune build @fmt
 
+
+new-openbsd:
+	ulimit -s 10000
+	doas ftp -o /usr/local/bin/shasum https://fastapi.metacpan.org/source/MSHELOR/Digest-SHA-6.02/shasum
+	doas chmod +x /usr/local/bin/shasum
+	doas mkdir -p /app/esy-install && \
+		doas chown -R $(USER):$(USER) /app/esy-install
+	gmake -E "APP_ESY=/app/esy" -E "APP_ESYINSTAL=/app/esy-install/" new-docker 
+
+APP_ESY ?= /app/esy
+APP_ESY_INSTALL ?= /usr/local/
+APP_ESY_RELEASE ?= /app/_release
+
+new-docker:
+	doas mkdir -p $(APP_ESY)&& \
+		doas chown -R $(USER):$(USER) $(APP_ESY)
+	opam init -y --disable-sandboxing --bare && \
+	opam switch create esy-local-switch 4.10.2+flambda -y && \
+	opam repository add duniverse "https://github.com/dune-universe/opam-repository.git#duniverse"
+	cp -rfp esy.opam $(APP_ESY)
+	opam install . --deps-only -y
+	cp -rfp . $(APP_ESY)
+	git -C $(APP_ESY)/esy-solve-cudf apply static-linking.patch && \
+	git -C $(APP_ESY) apply static-linking.patch
+	opam exec -- dune build -p esy
+	opam exec -- dune build @install
+	opam exec -- dune install --prefix $(APP_ESY_INSTALL)
+	$(APP_ESY_INSTALL)/bin/esy i --ocaml-pkg-name ocaml --ocaml-version 4.10.1002-flambda && \
+	$(APP_ESY_INSTALL)/bin/esy b --ocaml-pkg-name ocaml --ocaml-version 4.10.1002-flambda && \
+	$(APP_ESY_INSTALL)/bin/esy release --static --no-env --ocaml-pkg-name ocaml --ocaml-version 4.10.1002-flambda
+	opam exec -- dune uninstall --prefix $(APP_ESY_INSTALL)
+	CXX=c++ yarn global --prefix=$(APP_ESY_INSTALL) --force add ${PWD}/_release
+	mv _release $(APP_ESY_RELEASE)
+#COPY --from=builder /usr/local /usr/local
```

Comment:
Nit: left over `COPY`s from the Dockerfile

Reply:
What it means Nit?

I didn't implement this COPY --from=builder since I don't know exactly what it does.
I guess that could be done in the Dockerfile.

Reply:
Oh, I was just referring to trivial, non-critical, feedback - nitpicking - to remove those comments.